src/emulator/CMakeLists.txt

```diff
@@ -1,10 +1,28 @@
 set(VITA3K_ORG_NAME "Vita3K")
 set(VITA3K_APP_NAME "Vita3K")
 
-set(VITA3K_VERSION $ENV{TRAVIS_BRANCH})
+if(APPLE)
```

Comment:
I don't think we need to switch on platform here.

We should be able to use similar logic for all platforms, and just get the things that change per CI provider up front.

Reply:
No is needed for separed travis mac and Windows/appveyor

Reply:
because i don't kinow if "set(VITA3K_VERSION "")" works for travis, it is not me have set "set(VITA3K_VERSION $ENV{TRAVIS_BRANCH})", so i have juste thinked is needed for travis